lib/interpreters/split_clustering.rb

```diff
@@ -246,6 +247,9 @@ def self.split_solve_core(service_vrp, job = nil, &block)
 
         # RECURSIVELY CALL split_solve_core AND MERGE SOLUTIONS
         solutions = sides.collect.with_index{ |side, index|
+          # This is a recursive function and service_vrp[:split_level] needs to be corrected because
+          # it is incremented within the children of the other branch ("left-side") of this level
+          service_vrp[:split_level] = split_level + 1 # This needs to be here!
```

Comment:
In a test case, by the end of the resolution, the split_denominators array : `[1, 2, 4, 8, 16, 4, 8, 16, 8]` but in the logic I had in mind it should be `[1, 2, 4, 8, 16]`. Which was the purpose of this PR: https://github.com/Mapotempo/optimizer-api/pull/415

Regardless of the position of the `split_level` increment

Reply:
We already chat about this -- since the progress bar increaments are really beautiful at the moment, we decided to leave it for the main ticket (which will probably be based on something else then number of subproblems).